bamboo/integration_tests/performance_tests/test_tools.py

```diff
@@ -1,24 +1,25 @@
+import csv
 import os
 import re
 import sys
 import time
 import unittest
 
-def test_model(model_name):
-  return '--model=model_zoo/tests/model_%s.prototext' % model_name
+def model_test(model_name):
+  return '--model=../model_zoo/tests/model_%s.prototext' % model_name
```

Comment:
These paths are relative to the `bamboo` subdirectory which is where `test_suite.sh` is located. That means this test will fail if run outside of the `bamboo` subdirectory. Is that ok?